source/deployment/guest-accounts.rst

```diff
@@ -0,0 +1,128 @@
+.. _guest-accounts:
+
+Guest Accounts (Beta) (E10 and E20)
+===================================
+
+Available in `Enterprise Edition E10 and higher <https://about.mattermost.com/pricing/>`__. 
+
+Guest Accounts are a new role within Mattermost Enterprise editions which allow organizations to collaborate with individuals (such as vendors and contractors) outside of their organization. The Guest Account feature controls access to channels and other members within the organization. 
+
+An organization may use guest accounts to work on a website project with resources from external design firm.  Guest accounts could also be used to collaborate with customers on a support issue.
+
+Guests can: 
+
+- Pin messages to channels
+- Use slash commands (with the exception of those used to invite members)
+- Favorite channels
+- Mute channels
+- Update their account settings
+
+Guests cannot: 
+
+- Discover public channels
+- Join open teams
+- Create direct messages or group messages with members who aren’t within the same channel
+
+Enabling Guest Accounts
+-----------------------------------
+To enable Guests Accounts: 
+
+1. Navigate to **System Console > Authentication > Guest Access**.
+2. Update **Enable Guest Access** to ”True”.
+3. **Whitelist domains that are acceptable for Guest Access** (Optional).  
```

Comment:
```suggestion
3. (Optional) **Whitelist domains that are acceptable for Guest Access**.  
```

Reply:
^caveat: guideline of where `Optional` is placed (beginning or end of list item) might have changed, but previously I had it at the start for clarity.

Reply:
Had some feedback that it was a bit confusing, but I am ok with keeping it consistent throughout and having a larger discussion on this.